README.md

```diff
@@ -322,6 +325,34 @@ If you have not previously set up a RadarBox receiver that you want to reuse, do
 7. Next, wait a minute or two for the service to restart and head over to ADSB Exchange's 
 [Feeder Status](https://www.adsbexchange.com/myip/) page from a PC on the same network as the feeder. Verify that your feeder is shown as registered and that ADSB Exchange is receiving your feed and MLAT data. You can also verify your feeder's performance at the [ADSB Exchange Feeder Map](https://map.adsbexchange.com/mlat-map/) by searching for your site name.
 
+# Enable ADSB Exchange clones
+This project supports a number of ADSB Exchange clones that arose after the sale of ADSB Exchange. Currently there is support for [adsb.fi](https://adsb.fi/), [ADSB.lol](https://adsb.lol/), [ADS-B One](https://adsb.one/), [airplanes.live](https://airplanes.live/), [Planespotters.net](https://www.planespotters.net/), [TheAirTraffic](https://theairtraffic.com/), [AvDelphi](https://www.avdelphi.com/), [HP Radar](https://hpradar.com/), [RadarPlane](https://radarplane.com/) and [Fly Italy ADSB](https://flyitalyadsb.com/). If you would like any new services adding, please create a PR adding the new service or if you do not know how then please [open an issue](https://github.com/ketilmo/balena-ads-b/issues/new) with your request.
+
+For these services, you currently do not need any login or API credentials so there is no need to make an account with them (although some of them do offer this) and no credentials to add in balenaCloud. However, you do have to selectively enable each service (or you can enable all of them, or all but ADSB Exchange).
+
+To enable all services, or all services apart from ADSB Exchange, you can use one of the following *Device Variables*:
+
+- `ENABLE_ALL=true`
+- `ENABLE_ALL_BUT_ADSX=true`
+
+Note that you can use `true`, `enable`, `enabled`, `1`, `y`, `yes` or `on` for the value, and capitalisation does not matter. If you use `ENABLE_ALL` this will enable ADSB Exchange and you will then need to add the `ADSB_EXCHANGE_UUID` and `ADSB_EXCHANGE_SITENAME` as described [in the section above](#enable-adsb-exchange).
```

Comment:
```suggestion
Note that you can use `true`, `enable`, `enabled`, `1`, `y`, `yes` or `on` for the value, and capitalisation does not matter. If you use `ADSB_EXCHANGE_ENABLE_ALL` this will enable ADSB Exchange and you will then need to add the `ADSB_EXCHANGE_UUID` and `ADSB_EXCHANGE_SITENAME` as described [in the section above](#enable-adsb-exchange).
```